apps/authentication/mfa/__init__.py

```diff
@@ -1,5 +1,5 @@
+from .custom import MFACustom
+from .face import MFAFace
 from .otp import MFAOtp, otp_failed_msg
-from .sms import MFASms
 from .radius import MFARadius
-from .custom import MFACustom
-from .face import MFAFace
\ No newline at end of file
+from .sms import MFASms
```

Comment:
The only difference between the two code snippets is that they differ in their usage. They're both using imports from different modules. As far as I can see, there are no obvious issues with this code snippet. You can safely run it without further verification.